lib/DocumentsFieldArray/DocumentsFieldArray.js

```diff
@@ -60,8 +62,31 @@ class DocumentsFieldArray extends React.Component {
     !value ? <FormattedMessage id="stripes-core.label.missingRequiredField" /> : undefined
   )
 
+  renderCategory = (i) => {
+    const { documentCategories, name } = this.props;
+    return (
+      <Row>
+        <Col xs={11}>
+          <FormattedMessage id="ui-licenses.placeholder.documentCategory">
+            {placeholder => (
+              <Field
+                component={Select}
+                dataOptions={documentCategories}
+                id={`${name}-category-${i}`}
+                label={<FormattedMessage id="stripes-erm-components.doc.category" />}
+                name={`${name}[${i}].atType`}
+                placeholder={placeholder}
+                value="no"
```

Comment:
There shouldn't be a hardcoded `value` here, because that forces the `Field` to _always_ have the value of `"no"` as opposed to whatever the user selected. What was the goal with this prop?